config/software/libiconv.rb

```diff
@@ -21,7 +21,6 @@
 default_version "1.14"
 
 dependency "patch" if solaris2?
-dependency "mingw-runtime" if windows?
```

Comment:
Actually - now I'm curious.  Where are we getting libiconv from?  Won't we need to copy/package that from the external mingw compiler (along with all the other shared libraries like libatomic etc.).